content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,421 @@
+---
+title: Pod Eviction
+content_type: concept
+weight: 50
+---
+
+Eviction is the process by which {{<glossary_tooltip text="pods" term_id="pod">}}
+in a {{<glossary_tooltip text="node" term_id="node">}} are
+terminated. 
+
+In Kubernetes, there are two types of pod eviction:
+
+  1. Graceful eviction
+  2. Node-level eviction ({{< glossary_tooltip text="kubelet" term_id="kubelet" >}}
+     eviction)
+
+## Graceful eviction {#graceful-eviction}
+
+{{< glossary_definition term_id="graceful-eviction" length="all" >}}
```

Comment:
Maybe just use the short form, and then add extra text that's similar to the rest of the glossary definition.